contracts/core/modules/ERC20RebalancingSetExchangeIssuanceModule.sol

```diff
@@ -0,0 +1,301 @@
+/*
+    Copyright 2019 Set Labs Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+pragma solidity 0.5.7;
+pragma experimental "ABIEncoderV2";
+
+import { ReentrancyGuard } from "openzeppelin-solidity/contracts/utils/ReentrancyGuard.sol";
+import { SafeMath } from "openzeppelin-solidity/contracts/math/SafeMath.sol";
+
+import { CommonMath } from "../../lib/CommonMath.sol";
+import { ExchangeIssuanceLibrary } from "./lib/ExchangeIssuanceLibrary.sol";
+import { ERC20Wrapper } from "../../lib/ERC20Wrapper.sol";
+import { IExchangeIssuanceModule } from "../interfaces/IExchangeIssuanceModule.sol";
+import { ITransferProxy } from "../interfaces/ITransferProxy.sol";
+import { ModuleCoreState } from "./lib/ModuleCoreState.sol";
+import { RebalancingSetExchangeIssuance } from "./lib/RebalancingSetExchangeIssuance.sol";
+
+
+/**
+ * @title ERC20RebalancingSetExchangeIssuanceModule
+ * @author Set Protocol
+ *
+ * The ERC20RebalancingSetExchangeIssuanceModule supplementary smart contract allows a user to send an ERC20 
+ * and atomically issue a RebalancingSet
+ */
+contract ERC20RebalancingSetExchangeIssuanceModule is
+    ModuleCoreState,
+    RebalancingSetExchangeIssuance,
+    ReentrancyGuard
+{
+    using SafeMath for uint256;
+
+    /* ============ State Variables ============ */
+
+    // Address and instance of Transfer Proxy contract
+    address public transferProxy;
+    ITransferProxy public transferProxyInstance;
```

Comment:
It appears from the ABDK review that these interfaces can also function as an address. So when we store the address and the instance we're just duping the data so maybe it makes sense to remove `transferProxy` and `exchangeIssuanceModule`.

Reply:
+1 that is correct